lib/daylight/associations.rb

```diff
@@ -23,6 +23,7 @@ def association_path(reflection)
 
       def call_remote(remoted_method, model, http_method, args)
         response = self.method(http_method).call(remoted_method, args)
+        response = JSON.parse(response.body) if http_method != :get
```

Comment:
so what's happening here is that `:get` behaves differently from patch and post. for these verbs you have to call `body` on the result and then parse the json.